<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>602</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="w_bg" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,0,1">
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="styleSheet">
         <string notr="true">QTreeWidget{
border: 1px solid grey ;  
border-radius:20px;
padding: 1px 12px 1px 12px;  
background-color:rgb(255,255,255)
}</string>
        </property>
        <column>
         <property name="text">
          <string>API_Type</string>
         </property>
         <property name="font">
          <font>
           <family>Perpetua</family>
           <pointsize>13</pointsize>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/name.png</normaloff>icons/name.png</iconset>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Time/Parameters</string>
         </property>
         <property name="font">
          <font>
           <family>Perpetua</family>
           <pointsize>13</pointsize>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/property.png</normaloff>icons/property.png</iconset>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_18" stretch="3,1">
          <item>
           <widget class="QTextEdit" name="textEditDll">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QTextEdit
{
border: 1px solid grey ;  
border-radius:5px;
padding: 1px 1px 1px 1px;  
background-color:rgb(255,255,255)
}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDll">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>35</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Dll</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/gear.png</normaloff>icons/gear.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="3,1">
          <item>
           <widget class="QTextEdit" name="textEditInjector">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QTextEdit
{
border: 1px solid grey ;  
border-radius:5px;
padding: 1px 1px 1px 1px;  
background-color:rgb(255,255,255)
}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnInjector">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>35</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Injector</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/syringe.png</normaloff>icons/syringe.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="3,1">
          <item>
           <widget class="QTextEdit" name="textEditTestCode">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QTextEdit
{
border: 1px solid grey ;  
border-radius:5px;
padding: 1px 1px 1px 1px;  
background-color:rgb(255,255,255)
}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnTestCode">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>35</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>App</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/open.png</normaloff>icons/open.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QPushButton" name="pushButton_info">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>10</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size:12pt;</string>
            </property>
            <property name="text">
             <string>Info</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEditInfo">
            <property name="styleSheet">
             <string notr="true">QTextEdit
{
border: 1px solid grey ;  
border-radius:20px;
padding: 1px 20px 1px 20px;  
background-color:rgb(255,255,255)
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QPushButton" name="pushButton_pm">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size:12pt;</string>
            </property>
            <property name="text">
             <string>Process Modules</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEditModules">
            <property name="styleSheet">
             <string notr="true">QTextEdit
{
border: 1px solid grey ;  
border-radius:20px;
padding: 1px 20px 1px 20px;  
background-color:rgb(255,255,255)
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1,3,0,0">
        <item>
         <widget class="QLabel" name="title_label">
          <property name="styleSheet">
           <string notr="true">font-family: &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>HUSTGuard</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_19">
          <item>
           <widget class="QToolButton" name="btnStart">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/icons/start.png</normaloff>:/icons/start.png</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnClear">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/clear.png</normaloff>icons/clear.png</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="label_all_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>✧ProcessName✧</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignBottom|Qt::AlignHCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelName">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>✧ProcessPriority✧</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignBottom|Qt::AlignHCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPriority">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>✧ProcessID✧</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignBottom|Qt::AlignHCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelId">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>223</width>
            <height>44</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
